### PR TITLE
match rotary direction to standard rotaryio

### PR DESCRIPTION
**Pull request by jfurcean**
This makes it so clockwise rotation is positive and counter-clockwise is negative to match the standard `rotaryio` library.

### CODE REVIEW

File: adafruit_seesaw/seesaw.py [hunk 1]
```diff
@@ -373,7 +373,7 @@ def encoder_position(self, encoder=0):
         """The current position of the encoder"""
         buf = bytearray(4)
         self.read(_ENCODER_BASE, _ENCODER_POSITION + encoder, buf)
-        return struct.unpack(">i", buf)[0]
+        return struct.unpack(">i", buf)[0] * -1
```
Comment (dhalbert) [inline adafruit_seesaw/seesaw.py:376]:
```suggestion
        return -struct.unpack(">i", buf)[0]
```

Comment (dhalbert) [reply adafruit_seesaw/seesaw.py:376]: Negating is faster than multiplying.

File: adafruit_seesaw/seesaw.py [hunk 2]
```diff
@@ -384,7 +384,7 @@ def encoder_delta(self, encoder=0):
         """The change in encoder position since it was last read"""
         buf = bytearray(4)
         self.read(_ENCODER_BASE, _ENCODER_DELTA + encoder, buf)
-        return struct.unpack(">i", buf)[0]
+        return struct.unpack(">i", buf)[0] * -1
```
Comment (dhalbert) [inline adafruit_seesaw/seesaw.py:387]:
```suggestion
        return -struct.unpack(">i", buf)[0]
```

**Review by dhalbert — CHANGES_REQUESTED** [review]
Thanks for doing this so fast. I vote for negating, because the interpreter will not necessarily optimize the multiplication.

**Comment by dhalbert** [comment]
Could you add a comment too, to each place, explaining why you're doing the negation?

**Comment by jfurcean** [comment]
I updated it to use negation instead of multiplication and added comments.

**Comment by dhalbert** [comment]
@ladyada Did you want to forget about doing this in the library? I think you indicated you would rather just have some products be backwards and change the firmware for future products.

So the question is whether to merge this or just forget it.

**Comment by ladyada** [comment]
i would like it in the example not the library please!

**Pull request closed without merging**